159 update nearVector nearObject example (#1173)
weaviate/weaviate-io: developers/academy/zero_to_mvp/102_queries_1/_snippets/academy.queries.nearobject.equivalent.nearvector.mdx

<Tabs groupId="languages">
<TabItem value="py" label="Python">

> To run the `nearVector` query, first add the full vector to your code as a variable. You can view and copy the full vector below.

Reviewer: ```suggestion
To run the `nearVector` query, you need to provide the full vector. You can view and copy the full vector below.
```

</TabItem>
<TabItem value="graphql" label="GraphQL">

> To run the nearVector query, first add the full MEERKAT_VECTOR to your variables dictionary. You can view the full vector below.

Reviewer: Is this possible in GraphQL?

Author: Updated as per slack
